app/components/Name/Name.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import "react-slideshow-image/dist/styles.css";
+import { Fade, Zoom } from "react-slideshow-image";
+
+export default function Name() {
+  const words = ["Hello, there!", "Привет!", "Ciao!", "Selam!", "こんにちは！"];
+  const names = ["Фаузи Нажжап", "فوزي نجار"];
```

Comment:
these variables should be held in containers.

In general a good rule of thumb to follow is that we should only use components for display purposes, all other logic should be put into containers.